index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Baroka'a Portofolio</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+  <body>
+    <header>
+      <nav class="nav-bar">
+        <img class="logo" src="images/image_geometry.png" alt="logo">
+        <img class="toolbar" src="images/ic_menu.png" alt="">
```

Comment:
- [ ] [OPTIONAL] While this is working just fine, It would be better if you used an icon instead of an image for the hamburger menu. You can do that by using [Font Awesome](https://fontawesome.com/), which is a very handy tool to implement icons in your website.